MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/APICall.cs

```diff
@@ -25,9 +25,15 @@ public static IRestResponse GenericCall(string rest_request)
         }
         public static APOD AstronomyPhotoOfTheDay() 
         {
-            var response = GenericCall("apod");
+            var response = GenericCall("/planetary/apod?");
             var apod = JsonConvert.DeserializeObject<APOD>(response.Content);
             return apod;
         }
+        public static List<Photo> GetMarsRoverPhotosByDate(string earth_date)
```

Comment:
It *might* be helpful to have the date parameter as a DateTime object and convert it to a string here